packages/microlinkjs-wp/admin/index.php

```diff
@@ -0,0 +1 @@
+<?php // Silence is golden
```

Comment:
WTF

Reply:
Usually within a wordpress installation there'll be a `.htaccess`, restricting access to certain URLs such as plugin/theme/core files in Wordpress.

However, this _is_ optional and it's possible for a user to leave it out, therefore it's ideal to create these blank `index.php` files, which would make the page _blank_ if the user was to visit: www.mycoolwebsite.com/wp-content/plugins/microlinkjs-wp/admin

A bit of reference on this: http://shinephp.com/silence-is-golden/ (_lovely_ site 😆 )